chain/db/nanopay.go

```diff
@@ -0,0 +1,44 @@
+package db
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/nknorg/nkn/common"
+	"github.com/nknorg/nkn/common/serialization"
+)
+
+type nanoPay struct {
+	balance common.Fixed64
+	limit   uint32
```

Comment:
What about rename limit -> expiresAt or expiration? When limit occurs together with balance, it sounds like amount limit.

Reply:
done